### DISCUSSION TITLE
Search notes by parents attributes?

### DISCUSSION THREAD

**Discussion by nicollasricas** [TriliumNext/Trilium#3758] [Q&A] [answered]
Hi,

I'm trying to search for notes based on parent attributes. Is there a way to do this without a search script?

Tree:
```
Parent (#aaa)
	Child 1
		Child 1.5
	Child 2
		Child 2.1
		Child 2.2
		Child 2.3
	Child 3
		Child 3.1

Parent (#bbb)
	Child 10
	...
	Child 15
	...
	Child 30
```

Query:
`note.parents.#aaa`

Expected notes:
`Child 1, Child 2, Child 3 from #aaa parent`

Thanks!

**Comment by zadam** [comment, marked as answer]
Yes, the syntax is `note.parents.labels.aaa`
